iac/provider-gcp/nomad-cluster/network/ingress.tf

```diff
@@ -0,0 +1,58 @@
+resource "google_compute_health_check" "ingress" {
+  name = "${var.prefix}ingress"
+
+  timeout_sec         = 3
+  check_interval_sec  = 5
+  healthy_threshold   = 2
+  unhealthy_threshold = 2
+
+  http_health_check {
+    port         = var.ingress_port.port
+    request_path = var.ingress_port.health_path
+  }
+}
+
+resource "google_compute_backend_service" "ingress" {
+  name = "${var.prefix}ingress"
+
+  protocol  = "HTTP"
+  port_name = var.ingress_port.name
+
+  session_affinity = null
+  health_checks    = [google_compute_health_check.ingress.id]
+
+  timeout_sec = 65
+
+  load_balancing_scheme = "EXTERNAL_MANAGED"
+  locality_lb_policy    = "ROUND_ROBIN"
+
+  backend {
+    group = var.api_instance_group
+  }
+}
+
+resource "google_compute_url_map" "ingress" {
```

Comment:
Can we get a better name here? I think we now have two load balancers, "ingress" and "orch_map", neither of their names help understand what they do. Maybe "traefik" and "direct"?

Reply:
I like ingress as its common name and it describes what it is. Yes, "orch_map" is, in my opinion, a mistake, as it no longer makes sense. Ideally, I would like to transition away from the current load balancer once the migration is complete/rename it to something like "ingress-sandboxes" or a similar name to distinguish better.

I don't like to call it Traefik, as we can switch the ingress backend at any time in the future, but I'm okay with you coming up with a better name.